### DISCUSSION TITLE
Invert tour

### DISCUSSION THREAD

**Discussion by johannafr1984** [mytourbook/mytourbook#1362] [Q&A]
Hello, is there a possibility to invert a tour (change start and endpoint)? I searched for it but couldn`t find anything. I would be very grateful for your help.

**Comment by FJBDev** [comment]
Johanna,

I dont think this feature exists in MT as of today.
Feel free to create a feature request for it.

In the meantime, you can also check out this [opensource project](https://github.com/patrovite/Course_Generator) where this feature is available.
